typedmodels/models.py

```diff
@@ -321,8 +321,8 @@ def say_something(self):
 
     objects = TypedModelManager()
 
-    type = models.CharField(
-        choices=(), max_length=255, null=False, blank=False, db_index=True
+    dtm_type = models.ForeignKey(
+        ContentType, null=True, blank=False, db_index=True, on_delete=models.PROTECT
```

Comment:
Why the change to `null=True`?

Reply:
ah my bad, artefact of the migration, should be set to False after removing the 'type' field